data/json/items/armor/ballistic_armor.json

```diff
@@ -927,43 +927,32 @@
+    "weight": "21545 g",
     "volume": "6 L",
-    "price": 190000,
-    "price_postapoc": 6000,
-    "to_hit": -3,
+    "price": "5000 USD",
+    "price_postapoc": "60 USD",
+    "to_hit": { "grip": "bad", "length": "short", "surface": "any", "balance": "uneven" },
     "bashing": 6,
-    "material": [ "lightceramic", "kevlar_layered" ],
+    "material": [ "ceramic", "kevlar_layered" ],
     "symbol": "[",
     "looks_like": "kevlar",
-    "color": "light_red",
+    "color": "yellow",
     "warmth": 12,
-    "material_thickness": 3,
-    "//": "Ceramic disks inside block addition of steel plating modification",
     "flags": [ "STURDY", "OUTER", "NO_REPAIR" ],
-    "armor": [ { "encumbrance": 8, "coverage": 95, "covers": [ "torso" ] } ]
-  },
-  {
-    "id": "dragonskinempty",
-    "repairs_like": "survivor_suit",
-    "type": "ARMOR",
-    "category": "armor",
-    "name": { "str": "lamellar Kevlar vest" },
-    "description": "A lamellar Kevlar vest, there are intricately cut voids usually filled with stitched in ceramic disks, you could repair the stitching if needed",
-    "weight": "2060 g",
-    "volume": "4 L",
-    "price": 1900,
-    "price_postapoc": 2000,
-    "to_hit": -3,
-    "bashing": 6,
-    "material": [ "kevlar_layered" ],
-    "symbol": "[",
-    "looks_like": "dragonskin",
-    "color": "light_red",
-    "warmth": 10,
-    "material_thickness": 2,
-    "flags": [ "STURDY", "OUTER" ],
-    "armor": [ { "encumbrance": 6, "coverage": 25, "covers": [ "torso" ] } ]
+    "//2": "Ceramic disks in real-life dragon skin are 6.4mm thick; the actual fabric itself wasn't intended to provide high-grade protection, and so it's quite thin here.",
+    "armor": [
+      {
+        "material": [
+          { "type": "kevlar_layered", "covered_by_mat": 100, "thickness": 4.4 },
+          { "type": "ceramic", "covered_by_mat": 90, "thickness": 6.4 }
+        ],
+        "encumbrance": 8,
+        "coverage": 90,
```

Comment:
I'm having a hard time finding pics of people actually wearing it but it looks like it goes to the hips which would put overall coverage closer to 95% (basically only under the armpits missing from what I saw).